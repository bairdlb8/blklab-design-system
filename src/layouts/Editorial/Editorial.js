import React from 'react'
import PropTypes from 'prop-types'

import './Editorial.scss'

const Editorial = ({className=''}) => {
  return <div>Editorial</div>
}

Editorial.propTypes = {}

export default Editorial
