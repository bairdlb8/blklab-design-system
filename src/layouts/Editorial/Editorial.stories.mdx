import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Editorial from './Editorial'

<Meta title="Layouts / Editorial" component={ Editorial } />

# Editorial

## Add some text here

export const Template = (args) => {
  return <Editorial {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Editorial } />
