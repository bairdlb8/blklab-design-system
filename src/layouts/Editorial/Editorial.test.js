import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import Editorial from './Editorial'

expect.extend(toHaveNoViolations)

describe('<Editorial />', () => {
  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Editorial />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Editorial />)

    expect(asFragment(<Editorial />)).toMatchSnapshot()
  })
})
