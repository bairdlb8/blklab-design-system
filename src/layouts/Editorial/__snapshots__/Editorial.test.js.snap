// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Editorial /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Editorial
  </div>
</DocumentFragment>
`;
